data/net/minecraft/world/level/gameevent/GameEventListenerRegistrar.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/world/level/gameevent/GameEventListenerRegistrar
+	METHOD <init> (Lnet/minecraft/world/level/gameevent/GameEventListener;)V
+		ARG 1 listener
+	METHOD ifEventDispatcherExists (Lnet/minecraft/world/level/Level;Lnet/minecraft/core/SectionPos;Ljava/util/function/Consumer;)V
+		ARG 1 level
+		ARG 2 pos
+		ARG 3 consumer
+	METHOD lambda$onListenerMove$1 (Lnet/minecraft/world/level/gameevent/GameEventDispatcher;)V
+		ARG 1 dispatcher
+	METHOD lambda$onListenerMove$2 (Lnet/minecraft/world/level/gameevent/GameEventDispatcher;)V
+		ARG 1 dispatcher
+	METHOD lambda$onListenerRemoved$0 (Lnet/minecraft/world/level/gameevent/GameEventDispatcher;)V
+		ARG 1 dispatcher
```

Comment:
Please remove these methods to pass the validation check. <sub>(note to self: investigate this further)</sub>

Reply:
Okay, bit of context these are all lambda parameters.

Reply:
As of https://github.com/ParchmentMC/Parchment/commit/bfaf4327a6400c88404031cf0f12684e4a6d41e5 this should no longer be an issue it seems,

Reply:
I've rebased locally, considering this resolved for now.